test/surface_test.py

```diff
@@ -1198,6 +1198,21 @@ def test_convert_alpha_SRCALPHA(self):
         finally:
             pygame.display.quit()
 
+    def test_convert_get_alpha(self):
+        """ converted()d surfaces loaded from png transparent style surfaces should return 255.
+        """
+        pygame.display.init()
+        try:
+            pygame.display.set_mode((640, 480))
+
+            s1 = pygame.Surface((100, 100), pygame.SRCALPHA, 32)
+            s1_converted = s1.convert()
+            self.assertEqual(s1_converted.get_flags() & SRCALPHA, SRCALPHA)
+            self.assertEqual(s1_converted.get_alpha(), 255)
```

Comment:
This currently fails using 'pygame 2.1.3.dev5 (SDL 2.0.20, Python 3.9.7)', py x64 on Win10

Reply:
Is this managed by SDL?